src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_validators.py

```diff
@@ -197,8 +197,9 @@ def base64_encoded_certificate_type(string):
     return cert_data
 
 def datetime_type(string):
-    """ Validates UTC datettime in format '%Y-%m-%d\'T\'%H:%M\'Z\''. """
-    date_format = '%Y-%m-%dT%H:%MZ'
+    ''' Validates UTC datettime in format '%Y-%m-%d\'T\'%H:%M:%S\'Z\''
+        or  '%Y-%m-%d\'T\'%H:%M\'Z\''. '''
+    date_format = '%Y-%m-%dT%H:%M:%SZ' if len(string) is 20 else '%Y-%m-%dT%H:%MZ'
```

Comment:
I want one point. Python recommend [ask forgiveness instead permission](http://stackoverflow.com/questions/12265451/ask-forgiveness-not-permission-explain). Try and catch exception is not treated as seriously as other languages, in python it is widely accepted as a normal control flow.